{"name": "Invalid credentials", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//input[@type='email']\"}\n  (Session info: chrome=114.0.5735.198); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n0   chromedriver                        0x0000000105323f48 chromedriver + 4226888\n1   chromedriver                        0x000000010531c4f4 chromedriver + 4195572\n2   chromedriver                        0x0000000104f60d68 chromedriver + 281960\n3   chromedriver                        0x0000000104f9bfc8 chromedriver + 524232\n4   chromedriver                        0x0000000104fd2c58 chromedriver + 748632\n5   chromedriver                        0x0000000104f8ff1c chromedriver + 474908\n6   chromedriver                        0x0000000104f90ef4 chromedriver + 478964\n7   chromedriver                        0x00000001052e559c chromedriver + 3970460\n8   chromedriver                        0x00000001052e96f0 chromedriver + 3987184\n9   chromedriver                        0x00000001052ef5b4 chromedriver + 4011444\n10  chromedriver                        0x00000001052ea2fc chromedriver + 3990268\n11  chromedriver                        0x00000001052c21c0 chromedriver + 3826112\n12  chromedriver                        0x0000000105306088 chromedriver + 4104328\n13  chromedriver                        0x00000001053061e0 chromedriver + 4104672\n14  chromedriver                        0x0000000105315f28 chromedriver + 4169512\n15  libsystem_pthread.dylib             0x00000001890e7fa8 _pthread_start + 148\n16  libsystem_pthread.dylib             0x00000001890e2da0 thread_start + 8\n\n", "trace": "  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/login.py\", line 27, in step_impl\n    email_ele = CommonMethods().find_element(context.driver, email_field)\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/Helpers/common_methods.py\", line 8, in find_element\n    return driver.find_element(list(locator.keys())[0], list(locator.values())[0])\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "When user enters invalid username and password", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//input[@type='email']\"}\n  (Session info: chrome=114.0.5735.198); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n0   chromedriver                        0x0000000105323f48 chromedriver + 4226888\n1   chromedriver                        0x000000010531c4f4 chromedriver + 4195572\n2   chromedriver                        0x0000000104f60d68 chromedriver + 281960\n3   chromedriver                        0x0000000104f9bfc8 chromedriver + 524232\n4   chromedriver                        0x0000000104fd2c58 chromedriver + 748632\n5   chromedriver                        0x0000000104f8ff1c chromedriver + 474908\n6   chromedriver                        0x0000000104f90ef4 chromedriver + 478964\n7   chromedriver                        0x00000001052e559c chromedriver + 3970460\n8   chromedriver                        0x00000001052e96f0 chromedriver + 3987184\n9   chromedriver                        0x00000001052ef5b4 chromedriver + 4011444\n10  chromedriver                        0x00000001052ea2fc chromedriver + 3990268\n11  chromedriver                        0x00000001052c21c0 chromedriver + 3826112\n12  chromedriver                        0x0000000105306088 chromedriver + 4104328\n13  chromedriver                        0x00000001053061e0 chromedriver + 4104672\n14  chromedriver                        0x0000000105315f28 chromedriver + 4169512\n15  libsystem_pthread.dylib             0x00000001890e7fa8 _pthread_start + 148\n16  libsystem_pthread.dylib             0x00000001890e2da0 thread_start + 8\n\n", "trace": "  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/login.py\", line 27, in step_impl\n    email_ele = CommonMethods().find_element(context.driver, email_field)\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/Helpers/common_methods.py\", line 8, in find_element\n    return driver.find_element(list(locator.keys())[0], list(locator.values())[0])\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/vignesh/PycharmProjects/BDD_Framework/venv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1688884806190, "stop": 1688884811247}, {"name": "Then login should fail", "status": "skipped", "start": 1688884811248, "stop": 1688884811248}], "start": 1688884806189, "stop": 1688884811248, "uuid": "fadae21a-91fe-43a4-ba13-8601cb23602f", "historyId": "89a048edcbc8248da26faa31dabfcee1", "fullName": "Check login feature: Invalid credentials", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "lucky"}, {"name": "feature", "value": "Check login feature"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}